src/controller/model/endpoint.ts

```diff
@@ -181,7 +184,7 @@ class Endpoint extends Entity {
     }
 
     public async write(
-        clusterKey: number | string, attributes: KeyValue, options?: Options
+        clusterKey: number | string, attributes: KeyValue, options?: Options, transactionSequenceNumber?: number
```

Comment:
Please move `transactionSequenceNumber` into the `options`.

Reply:
Moved. But some methods already has it as parameter. I didn't change their signature.